### ISSUE TITLE
DOMException: Failed to execute 'querySelectorAll' on 'Document': [...] is not a valid selector. 

### ISSUE THREAD

**Issue by johnd0e** [IITC-CE/ingress-intel-total-conversion#186]
If enable `[x] Pause on caught exception` in debugger then there are many similar exceptions with such selectors:
- `#chatcontrols a:contains('faction')` (chat.js)
- ~~`[s!='']:x` (jquery-3.3.1.min.js)~~
- ~~`*,:x` (jquery-3.3.1.min.js)~~

Perhaps it is normal, as jQuery seems to work without problems (I cannot be completely sure, as our jQuery copy is minified, so it's not so easy to debug it in place).

But even if all is ok - it would be good to change our code to not emit such exceptions, to make debugging simpler.

**Comment by AlfonsoML** [comment]
The two exceptions in jquery-3.3.1.min.js are internal to that library and I would say that it's not possible to get ride of them as long as jQuery is being used. In Firefox it seems that it's possible to put it in a "black box", but there are far too many caught exceptions besides jQuery.

**Comment by johnd0e** [comment]
> The two exceptions in jquery-3.3.1.min.js are internal to that library

Yes. And actually these are one-time exceptions, so I do not bother much.
But exception with `#chatcontrols a:contains('faction')` is really annoying 'cause it repeats multiple times.

Although selector itself is valid for jQuery, I still prefer to revise that code, as it disturbs debugging IITC.
